frame/staking/src/pallet/impls.rs

```diff
@@ -822,8 +829,11 @@ impl<T: Config> Pallet<T> {
 
 		Self::register_weight(T::WeightInfo::get_npos_voters(validators_taken, nominators_taken));
 
-		let min_active_stake: T::CurrencyBalance =
-			if all_voters.len() == 0 { 0u64.into() } else { min_active_stake.into() };
+		let min_active_stake: T::CurrencyBalance = if all_voters.len() == 0 {
```

Comment:
May be some explanation of why we return `MinNominatorBond` and not 0? 

I am still not sure though what should be the correct value to return here though. Since we have no voters, probably 0 is a more reasonable value (but also this case is going to be extremely unlikely to happen so probably not worth overthinking).